conftest.py

```diff
@@ -0,0 +1,27 @@
+import os
+
+import simtools.config as cfg
+
+
+try:
+    cfg.loadConfig()
+except FileNotFoundError:
+    os.environ["HAS_CONFIG_FILE"] = "0"
+    os.environ["SIMTEL_INSTALLED"] = "0"
+    os.environ["HAS_DB_CONNECTION"] = "0"
+else:
+
+    os.environ["HAS_CONFIG_FILE"] = "1"
+
+    # Checking whether sim_telarray is properly installed
+    simtelPath = cfg.get("simtelPath")
+    simtelBinPath = simtelPath + "/sim_telarray/bin/sim_telarray"
+    os.environ["SIMTEL_INSTALLED"] = "1" if os.path.exists(simtelBinPath) else "0"
+
+    # Checking whether there is DB connection
+    useMongoDB = cfg.get("useMongoDB")
```

Comment:
I think this is not a good enough test. I think we should actually test if we can read from the DB instead. Maybe we can use one of the simple applications to get a value of a parameters (like one of the unit tests)? If we put it in a try/except block it should do the trick, no?

Reply:
Isn't just creating a DatabaseHandler instance enough? It should crash if there is no DB connection, right?

Reply:
No, it is not enough. We can create it successfully even with a broken connection.

Reply:
I committed some changes. Now I use db.getModeParameters("north", "lst-1", "Current") to test whether the DB connection is fine.

Reply:
OK, looks good. Need to test creating the DatabaseHandler in various environments probably, but we can fix it later if we see issues.